README.md

```diff
@@ -72,4 +72,4 @@ Give a ⭐️ if you like this project!
 
 This project is [MIT](./LICENSE) licensed.
```

Comment:
- [ ] Kindly replace the bolilerplate text in your readme file with your information, this includes your github, twitter and LinkedIn information as well as the Project Name and the `Built With` section.
![image](https://user-images.githubusercontent.com/81932478/197569397-7cf0a053-82d3-4441-823b-75e85b964597.png)

![image](https://user-images.githubusercontent.com/81932478/197569474-68dacc24-25dc-49b8-89d7-e8aef065cbb8.png)

- [ ] Also instructions on how to run the project should be present i.e `git clone` `npm install`, `npm start` e.t.c